pcsx2/SIO/Memcard/MemoryCardFile.cpp

```diff
@@ -278,14 +278,11 @@ void FileMemoryCard::Open()
 
 		if (FileSystem::GetPathFileSize(fname.c_str()) <= 0)
 		{
-			// FIXME : Ideally this should prompt the user for the size of the
-			// memory card file they would like to create, instead of trying to
-			// create one automatically.
-
 			if (!Create(fname.c_str(), 8))
 			{
-				Host::ReportFormattedErrorAsync("Memory Card", "Could not create a memory card: \n\n%s\n\n",
-					fname.c_str());
+				Host::ReportErrorAsync(TRANSLATE_SV("MemoryCard", "Memory Card Creation Failed"),
+					fmt::format(TRANSLATE_FS("MemoryCard", "Could not create the memory card:\n{}"),
```

Comment:
Should "Memory Card" be uppercased? There are more cases where it's shown as memory card, I shall not repeat those.

Reply:
Unresolving this question, as all issues I had were automatically resolved during commits because GitHub.

Reply:
NOTE for myself or anyone for the future:
Sony wrote the "memory card" itself in lowercase in English and probably in Title Uppercasing for non-English languages (only confirmed for Spanish). However, the MEMORY CARD slot has the memory card part in full caps.